manifests/g/GnuPG/GnuPG/2.3.2/GnuPG.GnuPG.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created using YamlCreate.ps1 v1.1.7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: GnuPG.GnuPG
+PackageVersion: 2.3.2
+PackageLocale: en-US
+Publisher: The GnuPG Project
+PublisherUrl: https://gnupg.org/index.html
+PublisherSupportUrl: https://gnupg.org/faq/gnupg-faq.html
+PrivacyUrl: https://gnupg.org/privacy-policy.html
+Author: The GnuPG Project
+PackageName: GNU Privacy Guard
+PackageUrl: https://gnupg.org/ftp/gcrypt/binary/
+License: GPL-3.0
+LicenseUrl: https://www.gnu.org/licenses/gpl-3.0-standalone.html
+Copyright: Copyright © 1996-2021 Free Software Foundation, Inc.
+CopyrightUrl: https://www.gnu.org/
+ShortDescription: GNU Privacy Guard (GnuPG or GPG) is a free-software replacement for Symantec's PGP cryptographic software suite.
+Description: GnuPG is a complete and free implementation of the OpenPGP standard as defined by RFC4880 (also known as PGP). GnuPG allows you to encrypt and sign your data and communications; it features a versatile key management system, along with access modules for all kinds of public key directories. GnuPG, also known as GPG, is a command line tool with features for easy integration with other applications. A wealth of frontend applications and libraries are available. GnuPG also provides support for S/MIME and Secure Shell (ssh).
+Moniker: gnupg
+Tags:
+- pgp
+- crypto
+- OpenPGP
+- S/MIME
+- X.509
+- cryptography
+- sign
+- GPG
```

Comment:
```suggestion
- pgp
- crypto
- openpgp
- s/mime
- x.509
- cryptography
- sign
- gpg
```

Reply:
Is there a requirement for the case of the label here? Because there are some proper nouns and abbreviations in it.

Reply:
`Note: The best practice is to present these terms in all lower case with hyphens rather than spaces.` - from https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.0.0/defaultLocale.md